rum/getting-started.mdx

```diff
@@ -59,6 +59,75 @@ if(window.Middleware){
 
 Once RUM is configured, metrics should begin appearing on the RUM Dashboard in Middleware within a few minutes. For more information on the RUM Dashboards see [here](https://docs.middleware.io/docs/rum/rum-middleware).
 
-For any issues please reach out to our team in your Slack channel or through the help menu in your [Middleware Account](https://app.middleware.io/auth/login).
+# Using RUM
+
+## Performance Summary
+
+The Performance Summary provides insight into your top-level RUM performance data. Learn about your Core Web Vitals Metrics, the total errors incurred, and the total unique user sessions and pages visits within a certain period of time.
+
+![](https://lh7-us.googleusercontent.com/0VNW2N5_GdWBkfztTGMD0bAGwSyZhbxMG6Gga00CZ8QSae-05rHQ9NSC_Lwi7akpGrsV3NalAf_ab6KDCxNvpok5Jw2Mn5u4VTUNUNu6GNQHQebiD253eXJNa3x06B8bCPEXaOXruoK2F2tycmdu1C0)
+
+## Session & Replays
+
+The Session & Replays section provides insight into your top-level RUM Session data and the ability to drill into specific session details to easily diagnose issues and understand user behavior.
```

Comment:
The Session & Replays section provides insight into your top-level RUM Session data. Each Sessions is represented by relevant requests made by the client as well as back-end spans called throughout the Session.